### PR TITLE
Linkedin account extraction

### PR DESCRIPTION
**Pull request by kovakina**
Extracting linkedin accounts from profile edit URL

### CODE REVIEW

File: config/features.yaml [hunk 1]
```diff
@@ -75,3 +75,14 @@ github_accounts:
...
         tags: ['Github Account']
         emojis: ['ID_BUTTON']
+
+# Linkedin account extraction from profile edit url
+linkedin_accounts:
+        query_string: 'source_short:"WEBHIST" AND
+          url:"https://www.linkedin.com/in/" AND "/edit/"'
+        attribute: 'url'
+        store_as: 'found_account'
+        re: 'https://www.linkedin.com/in/([A-z-\d]{1,39})/edit/'
```
Comment (obsidianforensics) [inline config/features.yaml]: Looks like LinkedIn account names can be 5-30 characters long, with a language id at the end (unclear if this is in addition to 30 or included). Maybe change the re to {5,32} ?

https://www.linkedin.com/help/linkedin/answer/87/customizing-your-public-profile-url?lang=en

File: config/features.yaml [hunk 2]
```diff
@@ -75,3 +75,14 @@ github_accounts:
         aggregate: True
         tags: ['Github Account']
         emojis: ['ID_BUTTON']
+
+# Linkedin account extraction from profile edit url
+linkedin_accounts:
+        query_string: 'source_short:"WEBHIST" AND
+          url:"https://www.linkedin.com/in/" AND "/edit/"'
```
Comment (obsidianforensics) [inline config/features.yaml]: Limit the "/edit/" search to url field:
url:"https://www.linkedin.com/in/" AND url:"/edit/"'

File: config/features.yaml [hunk 3]
```diff
@@ -75,3 +75,14 @@ github_accounts:
...
+        attribute: 'url'
+        store_as: 'found_account'
+        re: 'https://www.linkedin.com/in/([A-z-\d]{1,39})/edit/'
+        aggregate: True
+        tags: ['Linkedin Account']
```
Comment (obsidianforensics) [inline config/features.yaml]: LinkedIn (capital I)

File: config/features.yaml (hunk 1 above)
Comment (kovakina) [reply config/features.yaml]: Done

File: config/features.yaml (hunk 2 above)
Comment (kovakina) [reply config/features.yaml]: overlooked that. done!

**Comment by obsidianforensics** [comment]
LGTM. I think @berggren needs to approve this to merge it?

File: config/features.yaml [hunk 4]
```diff
@@ -75,3 +75,14 @@ github_accounts:
...
+
+# Linkedin account extraction from profile edit url
+linkedin_accounts:
+        query_string: 'source_short:"WEBHIST" AND
+          url:"https://www.linkedin.com/in/" AND url:"/edit/"'
+        attribute: 'url'
+        store_as: 'found_account'
+        re: 'https://www.linkedin.com/in/([A-z-\d]{5,32})/edit/'
+        aggregate: True
+        tags: ['LinkedIn Account']
```
Comment (berggren) [inline config/features.yaml]:
```suggestion
        tags: ['linkedin-account']
```

Comment (berggren) [reply config/features.yaml]: we are in the process of standardize how we handle tags and will move to kebab-case. Let's change this one to "linkedin-account"

**Pull request merged**